changelogs/2023-07/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: What's New in July 2023
+date: July 1, 2023
+author: Marc Paquette
+---
+
+This month's new features give admins better insight into their Coder instance and give developers support for Developer Containers and OpenStack.
+
+## Grafana Dashboard
+
+![Grafana dashboard](./static/grafana-dashboard.png)
+
+[Scaling](https://coder.com/docs/v2/latest/admin/scale) Coder is easier to monitor with support for [Grafana](https://grafana.com/). Coder can already generate [Prometheus metrics](https://coder.com/docs/v2/latest/admin/prometheus). Now you can keep an eye on these metrics in a Grafana dashboard.
```

Comment:
Let's re-word this to say that we provide a sample Grafana dashboard that customers can import. It contains important Prometheus metrics around scaling Coder and measuring adoption.